bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -3,10 +3,21 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Basic Install
+    certified: "false"
+    categories: "Integration & Delivery"
```

Comment:
Possibly expand the categories? E.g. include Networking & Messaging? (What does 'Delivery' mean in this context?)

Reply:
I have chosen this particular category based on the existing ones available at the Operator Hub.
But I believe we need to decide which one is the most appropriate to be used here.

Reply:
From interconnect-operator we had categories -> "Networking, Streaming & Messaging"

Suggest: "Integration, Networking, Streaming, Messaging"

Reply:
Here is the source for the allowed categories:
https://github.com/operator-framework/community-operators/blob/master/categories.json

These categories above can be defined as a comma separated list of strings.

Can we go with: "Integration & Delivery, Networking, Streaming & Messaging" ?

Reply:
+1